Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM golang:1.10-stretch
+FROM golang:1.11-stretch
+ENV GO111MODULE=on
 WORKDIR /go/src/github.com/JumboInteractiveLimited/Gandalf
+COPY go.mod ./
```

Comment:
Should also include go.sum here otherwise the versions will change across builds.